src/test/scala/org/apache/spark/sql/catalyst/optimizer/RangerSparkAuthorizerExtensionTest.scala

```diff
@@ -35,6 +36,20 @@ class RangerSparkAuthorizerExtensionTest extends FunSuite {
     assert(extension.apply(newPlan) === newPlan)
   }
 
+  test("ranger rule can not be excluded via a set command") {
```

Comment:
Could you add test for `set spark.sql.optimizer.excludedRules;`, the value mixed with ranger and non-ranger ones , and comma-separated ranger values?